specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-03-01-preview/examples/createSubscription.json

```diff
@@ -0,0 +1,35 @@
+{
+  "parameters": {
+    "billingAccountName": "73f8ab6e-cfa0-42be-b886-be6e77c2980c",
+    "api-version": "2018-03-01-preview",
+    "body": {
+      "properties": {
+        "offerType": "MS-AZR-0017P",
+        "displayName": "Test Ea Azure Sub",
+        "owners": [
+          {
+            "tenantId": "73f7ab6e-cfa0-42be-b506-be6e77c2700c",
```

Comment:
Let's also change this to 05053bdf-7e6a-4f96-854f-1dae531c170b - we likely won't allow owners from different tenants to be assigned owner on the subscription.